### PR TITLE
Validation: Fixed image-alt-text-duplicated issues

### PR DESCRIPTION
**Pull request by JustPies**
Fixed:

- image-alt-text-duplicated errors (duplicate alt text descriptions within an article)

This PR fixes 39 issues across 5 files and is part of ongoing Validation cleanup efforts. No other content has been edited or added.

### CODE REVIEW

File: dynamics-gp/financials/receivables-management-part5-utilities-routines.md [hunk 1]
```diff
@@ -1131,7 +1131,7 @@ Statements)
 2. Choose E-mail Options to open the E-mail Statements Options or E-mail
     Reprint Statements Options window.
 
-    ![A screenshot of a cell phone Description automatically generated](media/e755fb09b00080dfe7eadc23c24d9075.jpg)
+    ![Screenshot of window, showing selections of send email customer statements and long form.  Statement is entered in the E mail subject line.](media/e755fb09b00080dfe7eadc23c24d9075.jpg)
```
Comment (vatmicro) [inline dynamics-gp/financials/receivables-management-part5-utilities-routines.md:1134]: The word E-mail is missing a dash.

File: dynamics-gp/financials/receivables-management-part3-transaction-activity.md [hunk 2]
```diff
@@ -460,7 +460,7 @@ Use the Customer/Vendor Relationships window to link a customer to a vendor or a
 
 1. Open the Customer/Vendor Relationships window. (Sales \>\> Cards \>\> Customer/Vendor)
 
-   ![1e7d912d783aeb7fb89d71205efe68a4.jpg](media/1e7d912d783aeb7fb89d71205efe68a4.jpg)
+   ![Screenshot of Customer/Vender relationships window, before information has been input.](media/1e7d912d783aeb7fb89d71205efe68a4.jpg)
```
Comment (vatmicro) [inline dynamics-gp/financials/receivables-management-part3-transaction-activity.md:463]: For Contributor Guide tense guidelines, 

Change, "before information has been input", to 

"...before information is input."

File: dynamics-gp/financials/receivables-management-part3-transaction-activity.md [hunk 3]
```diff
@@ -496,7 +496,7 @@ Use the Customer/Vendor Consolidations window to create consolidation documents
 
 1. Open the Customer/Vendor Consolidations window. (Sales \>\> Transactions \>\> Customer/Vendor Trx.)
 
-    ![91622f2488ad897b2334f1007d0d7e36.jpg](media/91622f2488ad897b2334f1007d0d7e36.jpg)
+    ![Screenshot of Customer/Vendor Consolidations window, showing default entries and empty input boxes.](media/91622f2488ad897b2334f1007d0d7e36.jpg)
```
Comment (vatmicro) [inline dynamics-gp/financials/receivables-management-part3-transaction-activity.md:499]: Use "a" or "the" before the word "Customer" for this and all screenshots in the PR.

**Comment by JustPies** [comment]
Global effort to fix validation errors

@edupont04

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Review by edupont04 — APPROVED** [review]

**Pull request merged**